chatops-lark/pkg/events/handler/root.go

```diff
@@ -104,82 +114,238 @@ func NewRootForMessage(respondCli *lark.Client, cfg map[string]any) func(ctx con
 	cacheCfg := bigcache.DefaultConfig(10 * time.Minute)
 	cacheCfg.Logger = &log.Logger
 	cache, _ := bigcache.New(context.Background(), cacheCfg)
-	h := &rootHandler{Client: respondCli, Config: cfg, eventCache: cache}
+
+	// Get bot name for detect mention bot in group chat
+	// Note: In Lark API, @bot will be automatically converted to @_user_X format in message content
+	botName := ""
+	if botNameVal, ok := cfg["bot_name"]; ok {
+		if botNameStr, ok := botNameVal.(string); ok {
+			botName = botNameStr
+		}
+	}
+	if botName == "" {
+		botName = "Bot" // Default name
+	}
+
+	h := &rootHandler{Client: respondCli, Config: cfg, eventCache: cache, botName: botName}
 	return h.Handle
 }
 
 func (r *rootHandler) Handle(ctx context.Context, event *larkim.P2MessageReceiveV1) error {
 	eventID := event.EventV2Base.Header.EventID
+	r.logger = log.With().Str("eventId", eventID).Logger()
```

Comment:
should not update the `r.logger` every time in `Handle` call?

Reply:
If we want print the eventID, just do it in the other method, like `parseCommand` with local logger object rather than change the struct member.